### ISSUE TITLE
When input pattern has only one LU, put wblank in output automatically

### ISSUE THREAD

**Issue by khannatanmai** [apertium/apertium-recursive#66]
Shouldn't require a lemma clip when input pattern has only one LU. It should add the wordbound blank to the output automatically. This has been implemented in the 3-stage transfer system already.

**Comment by mr-martian** [comment]
Look at the instruction `LUCOUNT` to see how to count input LUs and then just check and set `wblank` outside the loop, I think.

**Comment by khannatanmai** [comment]
Yeah it was pretty trivial with normal transfer as well, just put it here as a reminder

तन्मय खन्ना
Tanmai Khanna

________________________________
